lib/ansible/modules/user.py

```diff
@@ -268,6 +268,11 @@
             - Requires O(local) is omitted or V(False).
         type: str
         version_added: "2.12"
+    account_inactive:
```

Comment:
its not really `account_inactive` as there are other ways to do that. The name should be more reflective of 'deactivation after password expiration' to avoid confusion. I know that the description is clear, but too many people will just take the name as the doc.

Reply:
I'll see what I can come up with. Naming things are the hardest part :/

Reply:
1000000% true

Reply:
Something along the lines of:
`password_expire_account_disable`
`password_account_disable`